lib/tbd/psi.rb

```diff
@@ -1941,14 +1941,38 @@ def process(model = nil, argh = {})
             ids = windows.keys + doors.keys + skylights.keys
           end
 
-          unless ids.include?(i)
-            log(ERR, "Orphaned subsurface #{i} (mth)")
+          adj = nil
```

Comment:
Interzone subsurfaces between CONDITIONED vs UNCONDITIONED spaces (e.g. trap door in an attic floor) is a rare - but not odd - occurrence. TBD wasn't catching these cases properly, raising an ERROR in the process. This is now caught correctly.